### ISSUE TITLE
Предложение 

### ISSUE THREAD

**Issue by morkvk** [kutovoys/marzban-torrent-blocker#1]
Не могли бы добавить возможность вместо блокировки через ufw выключать пользователю подписку в marzban

**Comment by morkvk** [comment]
pump

**Comment by kutovoys** [comment]
Я помню)
Но пока не доходят руки.

**Comment by GawrAme** [comment]
up, we need that option sir

**Comment by kutovoys** [comment]
> up, we need that option sir

Yes, I understand. Unfortunately, no time to implement it yet. I'll try to do it in the next month.

**Comment by Johnnybegood24** [comment]
Для панели 3x-ui подойдёт? Что-то надо менять?

**Comment by kutovoys** [comment]
> Для панели 3x-ui подойдёт? Что-то надо менять?

Работает с любыми xray-based панелями. 
Нужны логи и возможно поправить regex под панель и юзернеймы.

**Comment by Johnnybegood24** [comment]
Можете сделать гайд под 3xui? Что, где и как надо изменить. Заранее спасибо 🙏

**Comment by kutovoys** [comment]
> Можете сделать гайд под 3xui? Что, где и как надо изменить. Заранее спасибо 🙏

Нет. Вся необходимая информация есть в readme

**Comment by kutovoys** [comment]
Взаимодействие с панелями можно организовать через прием вебхука от тблокера. Рекомендую n8n для подобного.
